src/main.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const client = require('prom-client');
+const http = require('http');
+const { exec } = require('child_process');
+const { ss } = require('./parsers');
+
+const host = process.env.SIDECAR_MONITOR_HOST || '';
+const port = process.env.SIDECAR_MONITOR_PORT || 8089;
+let timer, output;
+
+// Prometheus
+const registry = new client.Registry();
+
+const activeConnectionsGauge = new client.Gauge({
+    name: 'prisma_active_ws_connections', 
```

Comment:
Since this will be a general utility, let's change this name to `active_ws_connections`.